### ISSUE TITLE
[Bug] Cyclic Flute and Modded Mobs

### ISSUE THREAD

**Issue by INooBTooMuch** [PrinceOfAmber/Cyclic#2180]
Minecraft Version: 1.18.2

Forge Version: 40.1.80

Mod Version: 1.7.9

Single Player or Server: Single

Describe problem (what you were doing; what happened; what should have happened):
When I try to copy the ID of a Modded Mob (I only tested this 2 times, both times with a villager from [More Villagers](https://www.curseforge.com/minecraft/mc-mods/more-villagers)) and then try to teleport it the game will crash but not close. I can't alt f4 out of the game and then it will show the Close progarm or Wait, so I don't know if the log file will help.

Log file link:
[Log](https://gist.github.com/INooBTooMuch/f06a0b1728e70fa5a4a5e08c0b9823ce)

**Comment by Lothrazar** [comment]
yeah thats weird, log file didnt show any crashlogs or lines pointing to something. i did not get it to crash when i tested with morevillagers 3.3.2   
![movillag](https://github.com/Lothrazar/Cyclic/assets/621529/8c1876bc-1691-4830-8159-e93666727ab3)

**Comment by INooBTooMuch** [comment]
![person-shrugging_1f937](https://github.com/Lothrazar/Cyclic/assets/107572028/c7a0ebe2-8ff0-4899-8591-f724efe51536)
